modules/github-workqueue/README.md

```diff
@@ -0,0 +1,90 @@
+# GitHub Workqueue Module
+
+This Terraform module provisions a GitHub webhook service that receives GitHub events and enqueues them to a workqueue for processing. It acts as a bridge between GitHub webhooks and the workqueue service, allowing you to process GitHub events asynchronously.
+
+## Architecture
+
+The module creates a **Webhook Service** that:
+- Receives GitHub webhook events via HTTP (through a load balancer)
+- Validates webhook signatures for security
```

Comment:
Looks like we create a random secret in the module, would be good to document how to get access to it so you can wire it up on the GitHub side. At least it used to be the case that you also needed to go twiddle GH to wire the secret up, but maybe that's not the case anymore? Or put another way, is there any additional stuff that one needs to do besides running this TF, or does this "Just Work" ™️ once you launch it?